server/modules/auth/auth.controller.ts

```
@@ -0,0 +1,372 @@
+    const { email } = req.body;
+
+    if (!email) {
+      return next(new ErrorResponse("Please provide an email", 400));
+    }
+
+    const user = await User.findOne({ email });
+
+    if (!user) {
+      return next(new ErrorResponse("User not found", 404));
+    }
+    let token = await Token.findOne({ userId: user._id });
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'token' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)